bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/internal/clangd/ClangdMetadataDefaults.java

```diff
@@ -13,21 +13,27 @@
  *******************************************************************************/
 package org.eclipse.cdt.lsp.internal.clangd;
 
-import java.util.Optional;
+import java.util.stream.Collectors;
 
 import org.eclipse.cdt.lsp.clangd.ClangdMetadata;
+import org.eclipse.cdt.lsp.clangd.ClangdOptions;
 import org.eclipse.cdt.lsp.internal.clangd.editor.LspEditorUiMessages;
-import org.eclipse.cdt.utils.PathUtil;
-import org.eclipse.core.runtime.IPath;
 import org.eclipse.core.runtime.preferences.PreferenceMetadata;
+import org.osgi.service.component.annotations.Reference;
 
-final class ClangdMetadataDefaults implements ClangdMetadata {
+public final class ClangdMetadataDefaults implements ClangdMetadata {
```

Comment:
Missed `@Component` annotation here

Reply:
Right, and I also missed it in `ClangdOptionsDefaults`.